packages/lodestar/src/network/encoders/response.ts

```diff
@@ -0,0 +1,107 @@
+import {IResponseChunk} from "./interface";
+import {IBeaconConfig} from "@chainsafe/lodestar-config";
+import {ILogger} from "@chainsafe/lodestar-utils/lib/logger";
+import {Method, MethodResponseType, Methods, ReqRespEncoding, RpcResponseStatus} from "../../constants";
+import {decode, encode} from "varint";
+import {encodeResponseStatus, getCompressor, getDecompressor} from "./utils";
+import BufferList from "bl";
```

Comment:
what's the reason to use `BufferList` instead of `Buffer`?

Reply:
We are appending a lot of buffers, with Buffer we would have to allocate and copy new Buffer everytime, while BufferList keeps internal array of buffers

Reply:
@mpetrunic Should we also use BufferList in eth2RequestDecode?